.editorconfig

```diff
@@ -0,0 +1,4 @@
+# 2 space indentation
+[*.py]
+indent_style = space
+indent_size = 2
```

Comment:
I think `.editorconfig` should be added to `.gitignore`, not added to the repo, for two reasons:

1. We do not use `*.py` files anywhere inside the project
2. We already have rules for indentation, and those are defined by `standard` linter

Reply:
I don't really agree, because one thing is the linter rules, another thing is the editor config, those are two separated concerns. Without `.editorconfig` everytime I start coding I need to go to my editor config and change the size spacing to 2, instead of 4 (my default). With this file, I don't need to. Not sure if I'm the only one who needs to do it. If yes, then okay, I'll add it to `.gitignore`